WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -132,14 +136,36 @@ class ScanViewModel @Inject constructor(
         }
     }
 
+    private fun showScanFinishedMessage(scanStateModel: ScanStateModel) {
+        val threatsCount = scanStateModel.threats?.size ?: 0
+        val message = UiStringText(
+            when (threatsCount) {
+                0 -> htmlMessageUtils.getHtmlMessageFromStringFormatResId(
+                    R.string.scan_finished_no_threats_found_message
+                )
+
+                1 -> htmlMessageUtils.getHtmlMessageFromStringFormatResId(
+                    R.string.scan_finished_threats_found_message_singular,
```

Comment:
Suggestion (💡): Since this string will always be used with a `1` argument, being a `singular`. What if we remove the `%s` from within the string and replace that with a hardcoded `1` instead? I consider this way a bit more safer due to the fact that it will not be possible to mix this string afterwards with a value other that `1`, which I am guessing it is our intention.

Reply:
Great suggestion, this will indeed be safer. In 3feccfe.